### DISCUSSION TITLE
Could you please add ORJSON support for high performance?

### DISCUSSION THREAD

**Discussion by accredited-unicorn** [mirumee/ariadne#665] [Show and tell]
I'm suggesting that Ariadne borrow a snippet of performance enhancing code from FastAPI...

```python
try:
    import orjson
except ImportError:  # pragma: nocover
    orjson = None  # type: ignore


class ORJSONResponse(JSONResponse):
    media_type = "application/json"

    def render(self, content: Any) -> bytes:
        assert orjson is not None, "orjson must be installed to use ORJSONResponse"
        return orjson.dumps(content)
```

ORJSON has been [benchmarked to reduce framework overhead](https://www.techempower.com/benchmarks/#section=data-r20&hw=ph&test=query), and it has been benchmarked to [increase throughout on batches of multiple queries](https://www.techempower.com/benchmarks/#section=data-r20&hw=ph&test=query). Replacing the default `json` library with `orjson` resulted in multiple query throughput that beat Uvicorn, Starlette, and vanilla FastAPI.

The snipped above should also enhance Ariadne's performance in a meaningful and measurable way.

**Comment by rafalp** [comment]
We already have PR that adds optional support for customizable JSON library: https://github.com/mirumee/ariadne/pull/650
